### DISCUSSION TITLE
WW3 in estuaries

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#1219] [Q&A]
I wanted to know if anyone has experience using Wavewatch III to calculate waves in estuaries. I recently used WW3 in the Río de La Plata estuary (Buenos Aires, Argentina) and did not get good results. However, outside the estuary, WW3 works well. Should I recompile WW3 with a different SWITCH? Or should I use another model like SWAN?

![Estuary](https://github.com/NOAA-EMC/WW3/assets/78570105/e76d6c4f-a9f4-424f-aa3d-52eca9ec3f33)

Thank you in advance.

**Comment by aliabdolali** [comment]
well, many factors play role here, like the physics you are using and spatial and temporal resolutions, grid type, ...
Look at the following publications, one down to 20 m resolution to resolve wetlands in a bay, and other ones to capture hurricanes, also in Great Lakes. We have tested the coastal capabilities of WW3 on unstructured meshes, so I'd recommend to switch to unstr and activate coastal physics, use a high res DEM for your mesh.

Abdolali, A., Banihashemi, S., Alves, J. H., Roland, A., Hesser, T. J., Anderson Bryant, M., and McKee Smith, J.: Great Lakes wave forecast system on high-resolution unstructured meshes, Geosci. Model Dev., 17, 1023–1039

Abdolali A, Hesser TJ, Anderson Bryant M, Roland A, Khalid A, Smith J, Ferreira C, Mehra A and Sikiric MD (2022) Wave Attenuation by Vegetation: Model Implementation and Validation Study. Front. Built Environ. 8:891612. doi: [10.3389/fbuil.2022.891612](https://doi.org/10.3389/fbuil.2022.891612)

Abdolali A., Roland, A., Van Der Westhuysen, A., Meixner, J., Chawla, A., Hesser, T.,  Smith, J.M. and M. Dutour Sikiric (2020), Large-scale Hurricane Modeling Using Domain Decomposition Parallelization and Implicit Scheme Implemented in WAVEWATCH III Wave Model, Coastal Engineering, 157, 103656, https://doi.org/10.1016/j.coastaleng.2020.103656

There are more articles, but I'd go with these 3 first.

**Reply by aronroland** [reply]
Hi,  so in order to have some better understanding of your problems u may want to share your setup.

**Reply by fed1942** [reply]
Sorry about the lack of information. This is the content of the SWITCH file I am using:

F90 NOGRB NOPA LRB4 SHRD PR3 UQ FLX0 LN1 ST4 STAB3 NL1 BT1 DB1 MLIM TR0 BS0 IC0 REF1 XX0 WNT1 WNX1 CRT1 CRX1 IS0 O0 O1 O2 O3 O4 O5 O6 O7 O11 O14 O16 OMPG OMPX
There are 2 or 3 obsolete options, but I don't remember which ones they are.


Calculation grids:
Global grid (1.25x1 degrees) > South Atlantic grid (0.31x0.31 degrees) > Buenos Aires grid (8x8 km) > RdeLP grid (1x1 km)

Wind grids:
 720x361 (CFSRv2)
